src/pages/Home/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useFeaturedBanners } from '../../utils/hooks/useFeaturedBanners';
+import SliderSection from './components/SliderSection';
+import productsCategories from '../../utils/productsCategory';
+import CategorySection from './components/CategorySection';
+import ProductSection from './components/ProductSection';
+import productList from '../../utils/products';
+
+const SectionTitle = styled.div`
+  margin-top: 3rem;
+`;
+
+const Home = () => {
+  const featuredBanners = useFeaturedBanners().data?.results;
+  const handleSlider = (moveDirection) => {
+    const contentSlider = document.getElementById('contentSlider');
+    const childLength = contentSlider.firstChild.clientWidth;
+
+    if (moveDirection === 'left') {
```

Comment:
You could use an object in order to prevent missmatch in string param, its very common when not working with TypeScript, something like `MOVE_DIRECTION.LEFT`